index.js

```diff
@@ -14,14 +14,15 @@ module.exports = async function(url, options, next) {
 
   const opts = Object.assign({}, options);
   const acc = { url, options: opts, pages: [], hrefs: [] };
+  const orig = url;
   let prev;
   let res;
 
   while (url && typeof url === 'string' && prev !== url) {
     prev = url;
     acc.hrefs.push(url);
     res = await needle('get', url, opts);
-    url = await next(url, res, acc);
+    url = await next(orig, res, acc);
```

Comment:
doh! I had this exact code in here before. Then I renamed the variables and must have removed it!

Maybe should use `acc.url` instead?

Reply:
> Maybe should use acc.url instead?

Would that look like this?: `acc.url = url;` first, then we would do `url = await next(acc.url, res, acc);`

> edit: and thanks!

NP! I can see this being useful in a lot of places.

Reply:
> Maybe should use acc.url instead?

Updated... I didn't notice that it was already on `acc`.